src/usr/local/www/status_captiveportal.php

```diff
@@ -126,9 +172,9 @@
 		<table class="table table-striped table-hover table-condensed sortable-theme-bootstrap" data-sortable>
 			<thead>
 				<tr>
+					<th><?=gettext("Username")?></th>
 					<th><?=gettext("IP address")?></th>
 					<th><?=gettext("MAC address")?></th>
-					<th><?=gettext("Username")?></th>
```

Comment:
They still show 'unauthenticated' in the column and it looked nicer to me to have the usernames/vauchers on the left side. I'll switch it back.